app/views/DocumentListView/DocumentListView.jsx

```diff
@@ -32,15 +32,16 @@ class DocumentListView extends React.Component {
     const {actions, state} = this.props
     const data = state.documents[this.getContentKey()] || {}
     const oldData = oldProps.state.documents[this.getContentKey()] || {}
-    const {isDeleting} = data
+    const {isDeleting, error} = data
     const {isDeleting: wasDeleting} = oldData
 
     // Have we just deleted some documents?
     if (wasDeleting && !isDeleting) {
-      const message =
-        wasDeleting > 1
-          ? `${wasDeleting} documents have been deleted`
-          : 'The document has been deleted'
+      const message = error
```

Comment:
I'm personally not a big fan of chaining the ternary operator, but I guess the alternative here wouldn't be much prettier? (I'm happy with it if you are.)